src/nvim/tui/tui.c

```diff
@@ -775,26 +802,6 @@ static void unibi_goto(UI *ui, int row, int col)
   unibi_out(ui, unibi_cursor_address);
 }
 
-static void unibi_out(UI *ui, int unibi_index)
```

Comment:
No reason, likely a result of rebasing a very old commit. I've cleaned up the patch.